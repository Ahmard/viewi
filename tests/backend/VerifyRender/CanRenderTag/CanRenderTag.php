<?php

class CanRenderTagComponent extends BaseComponent
{
    public string $title = 'Wellcome to my awesome application';

    function __construct()
    {
    }
}
