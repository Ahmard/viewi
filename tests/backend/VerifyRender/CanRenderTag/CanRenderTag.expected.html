<div>Wellcome to my awesome application</div>